circom/scripts/run_circuit.sh

```diff
@@ -24,11 +24,11 @@ else
 fi
 
 if [[ $(uname) == "Linux" ]]; then
-    TIMECMD='/usr/bin/time -f "Real time (seconds): %e\nMaximum resident set size (bytes): %M" -o'
+    TIMECMD='/usr/bin/env time -f "Real time (seconds): %e\nMaximum resident set size (bytes): %M" -o'
     STATCMD='stat --printf="%s" '
     OS="Linux"
 elif [[ $(uname) == "Darwin" ]]; then
-    TIMECMD="/usr/bin/time -h -l -o"
+    TIMECMD="/usr/bin/env time -h -l -o"
```

Comment:
This does not work. I replaced it with `/usr/bin/time` and it worked

Reply:
Maybe the best solution here is to make a TIMECMD environment variable, and set it to `/usr/bin/env time` in Nix, and set it to `/usr/bin/time` otherwise. I'll see about making that happen.

Reply:
I pushed a change. How's that?

Reply:
This still does not work using Nix on Mac. If I change it to `/usr/bin/time` and run it through Nix, then it works.

So, it turns out that Nix uses a GNU version of time:

```
[nix-develop-zk-Harness:] /usr/bin/env which time
/nix/store/crjc924qn35syvq9z0bsnkq2m8m97wzb-time-1.9/bin/time
[nix-develop-zk-Harness:] which /usr/bin/time
/usr/bin/time
```

Can you check this [commit](https://github.com/zkCollective/zk-Harness/commit/64549ed94f9e3c92db8a9b7a341b224c44ab4f30), and if you agree with that, include it in this PR?

Reply:
Hi, @StefanosChaliasos . Ah, I understand now. I reviewed your proposed fix. It looks good to me and works on my end. I trust that it works for you also.